pubspec.yaml

```diff
@@ -1,19 +1,20 @@
 name: test_coverage
 description: Command line utility to run tests in Dart VM and collect coverage data.
-version: 0.2.3
+version: 0.2.4
 homepage: https://github.com/pulyaevskiy/test-coverage
 author: Anatoly Pulyaevskiy <anatoly.pulyaevskiy@gmail.com>
 
 environment:
-  sdk: '>=2.0.0-dev.66.0 <3.0.0'
+  sdk: '>=2.3.0 <3.0.0'
```

Comment:
Why so strict? Doesn't look like we depend on something introduced in `2.3.0`. Maybe relax to `2.0.0`?

Reply:
We are moving to gating new language features based on the minimum supported SDK, so for example, to use the upcoming not null by default type annotations will require that you opt in by setting a high enough minimum SDK value. I'm trying to figure out what it will take to get the community as a whole to start opting in to higher minimum SDK versions.

Reply:
Sounds good!